Prime Video/dist/metadata.json

```diff
@@ -8,30 +8,12 @@
     "en": "Enjoy exclusive Amazon Originals as well as popular movies and TV shows. Watch anytime, anywhere. Start your free trial.",
     "de": "Spielfilme und Serien online streamen, als Einzelabruf online leihen oder kaufen bei Prime Video, Amazons großer Video on Demand Online-Videothek."
   },
-  "url": [
-    "www.amazon.cn",
-    "www.amazon.in",
-    "www.amazon.co.jp",
-    "www.amazon.com.sg",
-    "www.amazon.com.tr",
-    "www.amazon.ae",
-    "www.amazon.fr",
-    "www.amazon.co.uk",
-    "www.amazon.de",
-    "www.amazon.it",
-    "www.amazon.nl",
-    "www.amazon.es",
-    "www.amazon.ca",
-    "www.amazon.mx",
-    "www.amazon.com",
-    "www.amazon.com.au",
-    "www.amazon.com.br",
-    "www.primevideo.com"
-  ],
-  "version": "1.0",
+  "url": "www.primevideo.com",
```

Comment:
Seems unnecessary (the url) If using Regex

Reply:
you need a default url if for some reason the regex gives out an error

Reply:
Interesting